src/components/Chip/Chip.tsx

```diff
@@ -1,19 +1,32 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { highlightSubString } from "../../utils";
+import type { KeyboardEvent, MouseEvent } from "react";
+
+export type Props = {
+  lead?: string;
+  onClick: (
```

Comment:
I think this might be optional

Reply:
Yeah I think you're right, I've updated it.